### DISCUSSION TITLE
building qurakus-cxf on PPC64le(Power) Architecture

### DISCUSSION THREAD

**Discussion by shreyakajbaje** [quarkiverse/quarkus-cxf#1098] [Q&A]
Could anyone please share build steps and dependencies to build quarkus-cxf ?

**Comment by ppalaga** [comment]
This is a pretty standard Maven project. The _How to build_ section in the Contributor guide lists the build commands: https://docs.quarkiverse.io/quarkus-cxf/dev/contributor-guide/index.html#how-to-build 
Is there anything more you are looking for?

**Comment by shreyakajbaje** [comment]
Thanks @ppalaga for reply. Could you please share which version of java and mvn would be compatible?

**Reply by ppalaga** [reply]
Our CI uses Maven Wrapper, a.k.a. `mvnw` stored in the rood directory. That one is configured in `.mvn/wrapper/maven-wrapper.properties`

https://github.com/quarkiverse/quarkus-cxf/blob/d5a48426e2c0764bfed3d7062fe3b72ef839db7b/.mvn/wrapper/maven-wrapper.properties#L17

You can find the Maven version that is granted to work there. As you can see it is currently 3.9.5. If 3.9.5 is hard to get for you, then other recent 3.9 versions should work too.

For Java, the major version can be found in the root pom.xml 

https://github.com/quarkiverse/quarkus-cxf/blob/d5a48426e2c0764bfed3d7062fe3b72ef839db7b/pom.xml#L85

Our CI uses the Temurin distribution 

https://github.com/quarkiverse/quarkus-cxf/blob/d5a48426e2c0764bfed3d7062fe3b72ef839db7b/.github/workflows/build.yml#L31-L35
